etc/service/pd-server.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=PD is the abbreviation for Placement Driver. It is used to manage and schedule the TiKV cluster.
+Wants=network-online.target
+
+[Service]
+User=tidb
+ExecStart=/usr/bin/pd-server --config $CONFIGURATION_DIRECTORY/pd/config.toml
+ExecReload=/bin/kill -HUP $MAINPID
+Restart=on-failure
+KillSignal=SIGINT
+LimitNOFILE=82920
```

Comment:
Seems @tennix suggests we tune this down.

Reply:
@Hoverbear 65535 ?

Reply:
According to @tennix, only 1024.